src/core/account/lobby-api.ts

```diff
@@ -135,3 +136,32 @@ export async function makeLobbyApi(
 
   return out
 }
+
+async function unpackLoginRequest(
+  ai: ApiInput,
+  accountId: string,
+  lobbyId: string,
+  lobbyJson: EdgeLobbyRequest,
+  appId: string
+): Promise<EdgeLoginRequest> {
+  const info = await fetchAppIdInfo(ai, appId)
+
+  // Make the API:
+  const out: EdgeLoginRequest = {
+    appId,
+    displayName: info.appName,
+    displayImageUrl: info.imageUrl ?? info.lightImageUrl ?? info.darkImageUrl,
```

Comment:
Shouldn't we assume `displayImageUrl` is a lightmode image since that's how it was used prior? Hence I would suggest

```
    displayName: info.appName,
    displayImageLightUrl: info.imageUrl ?? info.lightImageUrl,
    displayImageDarkUrl: info.darkImageUrl,
    displayImageLightUrl: info.lightImageUrl ?? info.imageUrl
```

This would make any old apps that only use `displayImageUrl` on a light background continue to always work even if new appIds are added that only support `displayImageDarkUrl` and `displayImageLightUrl`

I would also be ok ditching `displayImageLightUrl` and assume `displayImageLightUrl` is light